sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
It's invalidated by default because stats info can be inaccurate (and in some cases very inaccurate), and can confuse regular users. At current stage it's better to be a feature for administrators and developers to see how cbo behaves in estimation. So I make the flag "internal".